### ISSUE TITLE
Static darafeed for React-Native app

### ISSUE THREAD

**Issue by augustofvilla** [tradingview/charting-library-examples#206]
Is there any way for send a static block of data to the library on a react-native app instead of access a remote datafeed?

**Comment by timocov** [comment]
@makedonsky94 can you please take a look?

**Comment by timocov** [comment]
Yes, I think this is possible. You need to implement your own datafeed and return the static data from it.

**Comment by DavitVosk** [comment]
Did you find a way how pass your custom datafeed to html file? I am injecting it, but on html side I get a string and not a class to be able to process it, please help
